amazon-kinesis-client/src/main/java/software/amazon/kinesis/leases/ShardSyncTask.java

```diff
@@ -72,7 +74,9 @@ public TaskResult call() {
         } catch (Exception e) {
             log.error("Caught exception while sync'ing Kinesis shards and leases", e);
             exception = e;
+            shardSyncSuccess = false;
         } finally {
+            MetricsUtil.addSuccess(scope, "SyncShards", shardSyncSuccess, MetricsLevel.DETAILED);
```

Comment:
Can we add a note on how customer will be using this metric for monitoring?

Reply:
Added